gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/binary/types/EdgeSerializer.java

```diff
@@ -72,12 +77,16 @@ protected Edge readValue(final Buffer buffer, final GraphBinaryReader context) t
     protected void writeValue(final Edge value, final Buffer buffer, final GraphBinaryWriter context) throws IOException {
 
         context.write(value.id(), buffer);
-        context.writeValue(value.label(), buffer, false);
+        // wrapping label into list here for now according to GraphBinaryV4, but we aren't allowing null label yet
```

Comment:
I'm assuming this is to prepare to a situation like no-label? I'm not sure this code is needed though.